src/OpenTelemetry.AutoInstrumentation.Native/integration_loader.cpp

```diff
@@ -55,8 +63,10 @@ void LoadIntegrationsFromFile(const WSTRING& file_path, std::vector<IntegrationM
     }
 }
 
-void LoadIntegrationsFromStream(std::istream& stream, std::vector<IntegrationMethod>& integrationMethods, const std::vector<WSTRING>& disabledIntegrationNames)
-{
+void LoadIntegrationsFromStream(
+    std::istream& stream, std::vector<IntegrationMethod>& integrationMethods,
```

Comment:
Since the previous method you put all parameters in their own line do the same here.
```suggestion
    std::istream& stream,
    std::vector<IntegrationMethod>& integrationMethods,
```